src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -203,7 +203,7 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::fullyCoupledSolverStep( real
     dtReturn = nonlinearImplicitStep( time_n, dt, cycleNumber, domain );
 
 
-    if( m_surfaceGenerator->solverStep( time_n, dt, cycleNumber, domain ) > 0 )
+    if( time_n >= 0 && m_surfaceGenerator->solverStep( time_n, dt, cycleNumber, domain ) > 0 )
```

Comment:
this is tricky, i suggest trying to use this flag https://github.com/GEOS-DEV/GEOS/blob/d3783ad15edb16510fed6f4eef214e01ff14ffde/src/coreComponents/physicsSolvers/multiphysics/PoromechanicsSolver.hpp#L232